sdcm/provision/azure/provisioner.py

```diff
@@ -157,12 +158,13 @@ def _vm_to_instance(self, v_m: VirtualMachine) -> VmInstance:
         tags = v_m.tags.copy()
         ssh_user = tags.pop("ssh_user")
         ssh_key = tags.pop("ssh_key")
+        creation_time = datetime.fromisoformat(tags.pop("creation_time", "1970-01-01 00:00:00"))
```

Comment:
why is `creation_time` set that old in the past? why not:

```suggestion
        creation_time = datetime.fromisoformat(tags.pop("creation_time", datetime.date.today()))
```

Reply:
This is the value that is used when there's no 'creation_time' tag in a VM, so we don't get error in list-resources command. I cannot provide N/A as this must be datetime object.
I picked some date to clearly show the date is invalid. Having set to today, it will confuse the user.

I'll fix list-resources to show N/A instead of this date.